walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,65 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.logging.Logger;
 
+/* FindMeetingQuery determines if people can have meetings based on each persons avaliable times */
 public final class FindMeetingQuery {
+
+  private static Logger log = Logger.getLogger(FindMeetingQuery.class.getName());
+
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+
+    Collection<TimeRange> originalAvailableTimes = Arrays.asList(TimeRange.WHOLE_DAY);
+    Collection<String> attendees = new ArrayList(request.getAttendees());
```

Comment:
don't forget to cover that case where there are events that do not involve the desired attendees

Reply:
i still don't see this case handled. are all your tests passing?